golang/cosmos/agoric/config.go

```diff
@@ -0,0 +1,35 @@
+package agoric
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// Bech32MainPrefix defines the Bech32 prefix used by all types
+	Bech32MainPrefix = "agoric"
+
+	// CoinType is used in the slip44 HD key derivation path.
+	// https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+	CoinType = 564
+
+	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address
+	Bech32PrefixAccAddr = Bech32MainPrefix
+	// Bech32PrefixAccPub defines the Bech32 prefix of an account's public key
+	Bech32PrefixAccPub = Bech32MainPrefix + sdk.PrefixPublic
+	// Bech32PrefixValAddr defines the Bech32 prefix of a validator's operator address
+	Bech32PrefixValAddr = Bech32MainPrefix + sdk.PrefixValidator + sdk.PrefixOperator
+	// Bech32PrefixValPub defines the Bech32 prefix of a validator's operator public key
+	Bech32PrefixValPub = Bech32MainPrefix + sdk.PrefixValidator + sdk.PrefixOperator + sdk.PrefixPublic
+	// Bech32PrefixConsAddr defines the Bech32 prefix of a consensus node address
+	Bech32PrefixConsAddr = Bech32MainPrefix + sdk.PrefixValidator + sdk.PrefixConsensus
+	// Bech32PrefixConsPub defines the Bech32 prefix of a consensus node public key
+	Bech32PrefixConsPub = Bech32MainPrefix + sdk.PrefixValidator + sdk.PrefixConsensus + sdk.PrefixPublic
+)
```

Comment:
I’ve seen these elsewhere. We should have a reference to the source-of-truth somewhere.

Reply:
Discussed with Kris and the equivalent code on the JS side lives in several places. Since we can't put a coherent reference here at the moment, we'll leave it as a TODO for the JS side to consolidate, then cross-reference.